core/src/apps/bitcoin/sign_tx/approvers.py

```diff
@@ -118,6 +118,7 @@ async def add_external_output(
         self,
         txo: TxOutput,
         script_pubkey: bytes,
+        index: int | None = None,
```

Comment:
Does it do anything? External output numbering already works on TT in master, I suppose the changes in `core/src/apps/bitcoin/sign_tx/` are leftover from different implementation of it that you had before?

Reply:
Right, these are from the past implementation. Deleted in [1ca9aa9](https://github.com/trezor/trezor-firmware/pull/2610/commits/1ca9aa977f19954db7544a87880b74de899af61d)